stable/artifactory/README.md

```diff
@@ -1,1160 +1,106 @@
-```
-
-If you're using Artifactory as SSO provider (e.g. with xray), you will need to have the following annotations, and change <artifactory-domain> with your domain:
-```yaml
-..
-    annotations:
-      kubernetes.io/ingress.class: nginx
-      nginx.ingress.kubernetes.io/configuration-snippet: |
-        proxy_pass_header   Server;
-        proxy_set_header    X-JFrog-Override-Base-Url https://<artifactory-domain>;
-```
-
-### Ingress additional rules
-
-You have the option to add additional ingress rules to the Artifactory ingress. An example for this use case can be routing the /xray path to Xray.
-In order to do that, simply add the following to a `artifactory-values.yaml` file:
-```yaml
-ingress:
-  enabled: true
-
-  defaultBackend:
-    enabled: false
-
-  annotations:
-    kubernetes.io/ingress.class: nginx
-    nginx.ingress.kubernetes.io/configuration-snippet: |
-      rewrite "(?i)/xray(/|$)(.*)" /$2 break;
-
-  additionalRules: |
-    - host: <MY_HOSTNAME>
-      http:
-        paths:
-          - path: /
-            backend:
-              serviceName: <XRAY_SERVER_SERVICE_NAME>
-              servicePort: <XRAY_SERVER_SERVICE_PORT>
-          - path: /xray
-            backend:
-              serviceName: <XRAY_SERVER_SERVICE_NAME>
-              servicePort: <XRAY_SERVER_SERVICE_PORT>
-          - path: /artifactory
-            backend:
-              serviceName: {{ template "artifactory.nginx.fullname" . }}
-              servicePort: {{ .Values.nginx.externalPortHttp }}
+helm upgrade --install artifactory --namespace artifactory center/jfrog/artifactory
 ```
+5. [Customize the product configuration](https://www.jfrog.com/confluence/display/JFROG/Installing+Artifactory#InstallingArtifactory-ProductConfiguration) including database, Java Opts, and filestore. Unlike other installations, Helm Chart configurations are made to the values.yaml and are then applied to the system.yaml. 
+Follow these steps to apply the configuration changes.
+a. Make the changes to the values.yaml file. 
+b. Run the command.
 
-and running:
 ```bash
-helm upgrade --install xray center/jfrog/artifactory -f artifactory-values.yaml
+helm upgrade --install artifactory --namespace artifactory -f values.yaml
 ```
 
-### Dedicated Ingress object for replicator service
+7. Connect to Artifactory. 
+It may take a few minutes for Artifactory's public IP to become available. Follow the instructions that are output by the install command above to get the Artifactory IP to access it. Below you will find a sample instruction of what to look for to pick the URL to reach Artifactory (in the example below, art77 is the release name and art is the namespace).
 
-You have the option to add additional ingress object to the Replicator service. An example for this use case can be routing the /replicator/ path to Artifactory.
-In order to do that, simply add the following to a `artifactory-values.yaml` file:
+Congratulations. You have just deployed JFrog Artifactory.
```

Comment:
@chukka @aaleksan20 Should this section be in a block ? It comes as a single line

Reply:
Yes, there was something wrong with that section. I made some edits to clear it up.